tests/backends/tests.py

```diff
@@ -388,8 +387,19 @@ def test_no_interpolation_on_sqlite(self):
         # This shouldn't raise an exception
         query = "SELECT strftime('%Y', 'now');"
         connection.cursor().execute(query)
-        self.assertEqual(connection.queries[-1]['sql'],
-            str_prefix("QUERY = %(_)s\"SELECT strftime('%%Y', 'now');\" - PARAMS = ()"))
+        self.assertEqual(connection.queries[-1]['sql'], query)
+
+    @unittest.skipUnless(connection.vendor == 'sqlite',
```

Comment:
There is a separate test case `SQLiteTests` which contains SQLite specific tests - how do you feel about moving this there? In fact, I'd be in favor of moving all the backend specific tests to `tests/backends/test_<backend>.py`. Would you be +1 on that idea (as a separate issue)?